builtin/game/chat.lua

```diff
@@ -770,7 +770,7 @@ core.register_chatcommand("pulverize", {
 		core.log("action", name .. " pulverized \"" ..
 			wielded_item:get_name() .. " " .. wielded_item:get_count() .. "\"")
 		player:set_wielded_item(nil)
-		return true, "An item was pulverized."
+		return true, name .. "'s item was pulverized."
```

Comment:
Not sure about having my name displayed. I usually know who I am. Maybe the item name instead could be interesting ?